de.metas.business.rest-api/src/main/java/de/metas/rest_api/ordercandidates/request/JsonOLCandCreateRequest.java

```diff
@@ -51,29 +53,30 @@
 @Value
 public final class JsonOLCandCreateRequest
 {
-	JsonOrganization org;
+	private JsonOrganization org;
 
 	@ApiModelProperty( //
-			required = true, //
-			value = "This translates to `C_OLCand.externalLineId`.\n"
-					+ "`externalLineId` and `dataSourceInternalName` together need to be unique.")
+
+			allowEmptyValue = false, //
+			value = "This translates to <code>C_OLCand.externalLineId</code>.\n"
+					+ "<code>externalLineId</code> and <code>dataSource</code> together need to be unique.")
 	String externalLineId;
 
 	@ApiModelProperty( //
-			required = true, //
-			value = "This translates to `C_OLCand.externalHeaderId`.\n"
-					+ "`externalHeaderId` and `dataSourceInternalName` together denote the unique group of olCands that were added in one bulk.")
+			allowEmptyValue = false, //
+			value = "This translates to <code>C_OLCand.externalHeaderId</code>.\n"
+					+ "<code>externalHeaderId</code> and <code>dataSource</code> together denote the unique group of olCands that were added in one bulk.")
```

Comment:
i find \`monospaced\` easier on the eye than `<code>monospaced</code>`..why change it?

Reply:
I saw it in other examples and thought it was preferred. Got rid of <code>  in the whole class.